ci/code_checks.sh

```diff
@@ -378,4 +378,13 @@ if [[ -z "$CHECK" || "$CHECK" == "typing" ]]; then
 fi
 
 
+### Checking for standardized documentation ###
+echo "Checking which files in pandas/doc/source have standardized documentation"
```

Comment:
I think this check should go under the `### PATTERNS ###` further up this file.

Could you also follow the pattern those checks have?

Reply:
Fixed!